testing.sagebrush.master/Stepwat_Inputs/Input/rgroup.in

```diff
@@ -68,10 +68,10 @@
  sagebrush     0.320  1.0000   1  0.0500   8     1       0   1      1     0.10000   2  0      0       0    0     1    1.00  0.00 0  0.00  0 
  a.cool.forb   0.030  5.0000   1  0.0500   2     1       0   1      1     0.20000   4  0      0       0    0     1    1.00  0.00 0  0.24  0  
  a.warm.forb   0.020  3.0000   1  0.0500   2     1       0   1      1     0.20000   4  0      0       0    0     1    1.00  0.00 0  0.24  0  
- p.cool.forb   0.160  8.0000   1  0.0500   2     1       0   1      1     0.20000   4  0      0       0    0     1    1.00  0.50 0  0.24  0  
+ p.cool.forb   0.060  8.0000   1  0.0500   2     1       0   1      1     0.20000   4  0      0       0    0     1    1.00  0.50 0  0.24  0  
  p.warm.forb   0.040  3.0000   1  0.0500   2     1       0   1      1     0.20000   4  0      0       0    0     1    1.00  0.50 0  0.24  0   
- a.cool.grass  0.050  50.000   1  0.0500   2     1       0   1      1     0.30000   3  0      0       0    0     1    1.00  0.00 0  0.24  0  
- p.cool.grass  0.220  15.000   1  0.0500   2     1       0   1      1     0.30000   3  0      0       0    0     1    1.00  0.80 0  0.24  0  
+ a.cool.grass  0.300  50.000   1  0.0500   2     1       0   1      1     0.30000   3  0      0       0    0     1    1.00  0.00 0  0.24  0  
+ p.cool.grass  0.070  15.000   1  0.0500   2     1       0   1      1     0.30000   3  0      0       0    0     1    1.00  0.80 0  0.24  0  
  p.warm.grass  0.040   5.000   1  0.0500   2     1       0   1      1     0.30000   3  0      0       0    0     1    1.00  0.80 0  0.24  0 
```

Comment:
These values were updated for testing purposes but need to be reverted before merging this branch into master.

Reply:
resolved by https://github.com/DrylandEcology/STEPWAT2/pull/481/commits/6b2e3c5f6c05eadaebeaa70ea2f6836cc0f9b3d2